### DISCUSSION TITLE
Example App

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2437] [General]
This is the 365 admin interface. How can I get this theme for my entire application?

![image](https://github.com/user-attachments/assets/6f79f7a2-361e-46e7-bd94-1da7854a76b1)

**Comment by vnbaaij** [comment]
We don't have something out of the box for that. 

The example at https://www.fluentui-blazor.net/Layouthttps://www.fluentui-blazor.net/Layout + the code from the demo site should give you some handles to built something like it. I say 'like' because for example the nav menu in that site looks different with the active bar closer to the item. For us it is always on the left (or right) margin.

**Reply by franklupo** [reply]
the current theme lacks visual depth
